config/initializers/breakers.rb

```diff
@@ -80,7 +80,8 @@
     IAMSSOeOAuth::Configuration.instance.breakers_service,
     CovidVaccine::V0::VetextConfiguration.instance.breakers_service,
     VEText::Configuration.instance.breakers_service,
-    PagerDuty::Configuration.instance.breakers_service
+    PagerDuty::Configuration.instance.breakers_service,
+    ClaimsApi::LocalBGS.breakers_service
```

Comment:
I see all of these have `.instance` but ours does not? Did we miss this or is it not needed?

Reply:
it depends, the `EVSS::ClaimsService` above doesn't - I did a `self.breakers_service` in the `LocalBGS` so I don't need `.instance` - I don't think there's any benefit either way for it? The `LocalBGS` initializer takes 2 args, I wonder if those would still be required if I did `.instance`.